app/plugins.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-dynamic-require */
```

Comment:
This would be better placed inline with the code it applies to. `eslint-disable-line` can be used or `eslint-disable-next-line`.